system/lib/gl/webgl1.c

```diff
@@ -74,14 +74,7 @@ EMSCRIPTEN_WEBGL_CONTEXT_HANDLE emscripten_webgl_create_context(const char *targ
   }
   pthread_once(&tlsInit, InitWebGLTls);
 
-  if (attributes->proxyContextToMainThread == EMSCRIPTEN_WEBGL_CONTEXT_PROXY_ALWAYS ||
-    (attributes->proxyContextToMainThread == EMSCRIPTEN_WEBGL_CONTEXT_PROXY_FALLBACK && !emscripten_supports_offscreencanvas())) {
```

Comment:
This check looks a little different than JS: it also has `emscripten_supports_offscreencanvas`. I don't know the offscreen canvas stuff well - is that not an issue?